superset/typing.py

```diff
@@ -57,7 +57,7 @@ class AdhocMetricColumn(TypedDict, total=False):
 class AdhocMetric(TypedDict, total=False):
     aggregate: str
     column: Optional[AdhocMetricColumn]
-    expressionType: str
+    expressionType: Literal["SIMPLE", "SQL"]
```

Comment:
I guess this is screaming out to be an enum. Definitely a future TODO.

Reply:
Good idea. In the next iteration I'll see if I can convert some of these `Literal`'s to `Enum`s. But it may well be that this is the most accurate representation of this type, as we want to indicate that this is a string that can only take on two values (equivalent of TypeScript's `{ expressionType: 'SIMPLE' | 'SQL' }`)